FEM/rf_pcs.cpp

```diff
@@ -3371,6 +3375,10 @@ void CRFProcess::VariableDynamics()
 	pcs_secondary_function_name[pcs_number_of_secondary_nvals] = "STRAIN_PLS";
 	pcs_secondary_function_unit[pcs_number_of_secondary_nvals] = "-";
 	pcs_secondary_function_timelevel[pcs_number_of_secondary_nvals] = 1;
+	pcs_number_of_secondary_nvals++;
```

Comment:
In that time, we did not image the significant increase of memory usage, and we did not have code review either. For deformation analysis, because of high order elements, the memory usage is huge for 3D problems is huge. Besides, unnecessarily memory also influences the computational efficiency.  

I will added if-condition for other variables later on.